### PR TITLE
Bug Fixes: Typeo in Remote Imports, Default Privileged mode for Docker Pipeline, Bundle Issue in Cross Account Stack Ref Fn

### PR DESCRIPTION
**Pull request by timpur**

### CODE REVIEW

File: packages/@cdk-cosmos/core/src/helpers/remote.ts [hunk 1]
```diff
@@ -180,11 +180,7 @@ export class RemoteCluster {
   static import(scope: Construct, id: string, exportName: string, vpc: IVpc): ICluster {
     const clusterName = Fn.importValue(`${exportName}Name`);
     const securityGroups = [
-      SecurityGroup.fromSecurityGroupId(
-        scope,
-        `${exportName}SecurityGroup`,
-        Fn.importValue(`${exportName}SecurityGroup`)
-      ),
+      SecurityGroup.fromSecurityGroupId(scope, `${id}SecurityGroup`, Fn.importValue(`${exportName}SecurityGroup`)),
```
Comment (jbnaik) [inline packages/@cdk-cosmos/core/src/helpers/remote.ts:183]: Will this cause name change of SecurityGroup...? I suppose not as snapshot hasn't changed.

File: packages/private/cross-account-stack-ref-handler/package.json [hunk 2]
```diff
@@ -16,15 +16,29 @@
...
   "scripts": {
-    "build": "parcel build src/index.ts --out-dir lib --target node --bundle-node-modules && yarn bundle",
+    "build": "parcel build src/index.ts --target node --cache-dir .cache && yarn bundle",
     "test": "jest",
     "format": "prettier --write **/*.ts !**/*.d.ts",
     "lint": "eslint . --ext .js,.ts",
     "clean": "rm -rf lib && rm -rf .cache",
     "bundle": "cd lib && rm -f bundle.zip && zip bundle.zip *"
   },
+  "targets": {
```
Comment (jbnaik) [inline packages/private/cross-account-stack-ref-handler/package.json:26]: Is there a way to test this lambda?

File: packages/@cdk-cosmos/core/src/helpers/remote.ts (hunk 1 above)
Comment (timpur) [reply packages/@cdk-cosmos/core/src/helpers/remote.ts:183]: only if your creating constructs under the portal

File: packages/private/cross-account-stack-ref-handler/package.json (hunk 2 above)
Comment (timpur) [reply packages/private/cross-account-stack-ref-handler/package.json:26]: um not really, we have tests, but the issue was a bundle not the code

**Pull request merged**